Lesson_9/Tusk_3.py

```diff
@@ -0,0 +1,16 @@
+fib_sequence = [0, 1, 1]
+
+
+def rec_fibonacci(number, list):
+    if len(list) < number + 1:
+        list.append(list[-1] + list[-2])
```

Comment:
You don't need a list here to store values. The recursive approach means that you will be able to use values from recursive calls